src/redux/books/bookSlice.js

```diff
@@ -12,6 +14,14 @@ export const getBooks = createAsyncThunk('book/getBooks', async () => {
   }
 });
 
+export const FetchBooks = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
```

Comment:
- [ ] Great work so far 👍 but it would be great if you use useEffect hook in the **BookListContainer** component to follow the project requirements as suggested by the previous reviewer 👍